docs/core/man/doveadm-fs.1.md

```diff
@@ -78,10 +78,6 @@ Store data at the path provided.
 :   Save provided hash as content hash. Must be either MD5 or SHA256 in
     hexdigits.
 
-**-m** *key=value*
-:   Assign a value to a metadata. Can be provided as many times as needed.
-    Requires either metawrap as driver or metadata capable FS-driver.
```

Comment:
This parameter is still there. It's not obsolete.

Reply:
I don't know where I got this from. But if it is in fact still there, I added it back to the SYNOPSIS.